apps/web/src/app/styles.css

```
@@ -1,37 +1,934 @@
+  --text-muted: #748197;
+  --accent: #347dff;
+  --accent-soft: #162e53;
+  --green: #4ac26b;
+  --amber: #d8a72d;
+  --offline: #6d7788;
+  --sidebar-left: 260px;
+  --sidebar-right: 358px;
+  --topbar-height: 64px;
+  --mono: "SFMono-Regular", "Cascadia Code", "Roboto Mono", Consolas, monospace;
+  --sans: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  background: var(--bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)